setup/catalogs/devops/prometheus.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: deployments.plural.sh/v1alpha1
+kind: PrAutomation
+metadata:
+  name: prometheus-setup
+spec:
+  name: prometheus-setup
+  icon: https://github.com/pluralsh/plural-artifacts/blob/main/monitoring/plural/icons/monitoring.png?raw=true
+  identifier: mgmt
+  documentation: |
+    Sets up a victoria metrics prometheus instance for metrics storage and an agent for shipping metrics.
+  creates:
+    git:
+      ref: john/prod-2967-observability-catalog-victoriametrics-prometheus
+      folder: catalogs/devops/prometheus
+    templates:
+    - source: README.md
+      destination: "documentation/prometheus/README.md"
+      external: true
+    - source: helm/agent.yaml.liquid
+      destination: "helm/monitoring/agent.yaml.liquid"
+      external: true
+    - source: helm/mgmt.yaml.liquid
+      destination: "helm/monitoring/mgmt.yaml.liquid"
+      external: true
+    - source: generatedsecret.yaml
+      destination: "bootstrap/apps/prometheus/generatedsecret.yaml"
+      external: true
+    - source: agent.yaml
+      destination: "bootstrap/monitoring/services/agent.yaml"
```

Comment:
we should make the destination of these two under `bootstrap/apps/prometheus` so they're next to the generated secret